sdk/how-to-guides/k8s-workload.md

```diff
@@ -472,6 +449,57 @@ A check is considered healthy until it's had `threshold` errors in a row (the de
 
 See the [layer specification](https://github.com/canonical/pebble#layer-specification) for more details about the fields and options for different types of checks.
 
+<a href="#heading--respond-to-check"><h3 id="heading--respond-to-check">Respond to a check failing or recovering</h3></a>
+
+[note status="version"]Ops 2.15 and Juju 3.6[/note]
```

Comment:
I think you have to close the note on the next line, or it doesn't render correctly on Discourse.